irr/src/OpenGL/Driver.cpp

```diff
@@ -198,24 +200,25 @@ COpenGL3DriverBase::~COpenGL3DriverBase()
 	}
 }
 
-void COpenGL3DriverBase::initQuadsIndices(int max_vertex_count)
+void COpenGL3DriverBase::initQuadsIndices(u32 max_vertex_count)
 {
-	int max_quad_count = max_vertex_count / 4;
-	std::vector<GLushort> QuadsIndices;
-	QuadsIndices.reserve(6 * max_quad_count);
-	for (int k = 0; k < max_quad_count; k++) {
+	u32 max_quad_count = max_vertex_count / 4;
+	u32 indices_size = 6 * max_quad_count;
+	if (indices_size == QuadIndexVBO.getSize() * sizeof(u16))
```

Comment:
Couldn't this also be `<=` to avoid some additional calculations?

Reply:
not clear if calling this multiple times even makes sense. we don't do that.